client/log_power.service

```diff
@@ -0,0 +1,8 @@
+[Unit]
+Description=Take and copy monitoring data of the 2024 power system
+
+[Service]
+User=rno-g
+Restart=always
+ExecStart=python3 /home/rno-g/micromini-firmware/client/log_power.py
+ExecReload=/bin/kill -USR1 $MAINPID
```

Comment:
do you do anything when you get a SIGUSR1? There's no configuration to reread or anything... I would just omit that line!

Reply:
Okay, this was copy past from one of yours.